proposed/extended-coding-style-guide.md

```diff
@@ -396,6 +396,8 @@ Property names MUST NOT be prefixed with a single underscore to indicate
 protected or private visibility. That is, an underscore prefix explicitly has
 no meaning.
 
+When present, type declaration MUST precede property name.
```

Comment:
I don't understand, where else can the type declaration be?

Reply:
`public int static $test`?

Reply:
Then you must be more precise:
```suggestion
When present, type declaration MUST immediately precede property name.
```

Reply:
Applied change. Actually I'm not sure such order is allowed by PHP. Had no time to check source code or get PHP 7.4 working locally. Anyone willing to do so?

Reply:
```
docker run --rm -it devilbox/php-fpm-7.4:latest php -r 'class Test { public int static $bar; }
```

results in

```
Parse error: syntax error, unexpected 'static' (T_STATIC), expecting variable (T_VARIABLE) in Command line code on line 1
```

Reply:
Thank you! Then the rule isn't needed at all.